pkg/harvester/detail/harvesterhci.io.host/HarvesterHostDisk.vue

```diff
@@ -99,6 +101,17 @@ export default {
     />
     <div v-if="!value.isNew">
       <div class="row">
+        <div class="col span-12">
+          <LabeledSelect
+            v-model="value.tags"
+            :mode="mode"
+            :multiple="true"
+            :taggable="true"
+            :options="[]"
```

Comment:
This dropdown, and the others in the pr with `:options="[]"`, are a little confusing to me. I know that user-defined options can be typed in, but I think this is the wrong component for an array of user-generated values: I think we should not be using LabeledSelect if there are never `options`. In other parts of the dashboard, an array of user-generated values is created with LabeledInput where the user enters a  comma-separated list, which is then formatted something like this:
```
:value="tags ? tags.join(', ') : '' ""
@input="input => tags = input.split(',').map(tag => tag.trim()).filter(tag => tag?.length)"
```

Reply:
Yes, the `:options="[]"` is confuse.  I try the LabeldInput, but it requires comma-separated, user need to add or remove comma to change the array. I want to use the tag which is look like the longhorn UI do. Do you have any idea or should I write a compont just same as longhorn UI?

https://user-images.githubusercontent.com/18737885/191061712-49de6e79-d538-4752-aa4a-d52ad12b5d5c.mov

Reply:
I like the Longhorn component but it would take time to write one for the Dashboard and I heard we want this PR merged soon so it can go in RC1. I made a small PR into your branch with a quicker solution for now: https://github.com/n313893254/dashboard/pull/7 

Feel free to reject this idea, we can revisit the tag component when there's more time. I think this is not a big issue and can be addressed in a later PR.

Reply:
The longhorn tag component will be tracked in https://github.com/harvester/harvester/issues/2837